src/main/java/core/basesyntax/model/User.java

```diff
@@ -8,6 +8,16 @@ public class User {
     private String password;
     private Integer age;
 
+    public User() {
+    }
+
+    public User(Long id, String login, String password, Integer age) {
+        this.id = id;
```

Comment:
You don`t have to initialize id fields. this happens when you add user to storage